binderhub/static/js/index.js

```diff
@@ -59,50 +59,33 @@ function v2url(providerPrefix, repository, ref, path, pathType) {
   return url;
 }
 
+function loadConfig(callback) {
+  var req = new XMLHttpRequest();
+  req.onreadystatechange = function() {
+    if (req.readyState == 4 && req.status == 200)
+      callback(req.responseText)
+  };
+  req.open('GET', BASE_URL + "_config", true);
+  req.send(null);
+}
```

Comment:
maybe we should do this request only 1 time in `indexMain` function and pass config value into a global variable, then `updateRepoText` function can use that global variable. what do you think?

Reply:
That's a good idea. Thanks @bitnik!